framework/src/executioners/Eigenvalue.C

```diff
@@ -88,6 +95,13 @@ Eigenvalue::init()
   _eigen_problem.getNonlinearEigenSystem().precondMatrixIncludesEigenKernels(
       getParam<bool>("precond_matrix_includes_eigen"));
 #endif
+  if (isParamValid("normalization"))
+  {
+    auto normpp = getParam<PostprocessorName>("normalization");
+    const auto & exec = _problem.getUserObject<UserObject>(normpp).getExecuteOnEnum();
+    if (!exec.contains(EXEC_LINEAR))
+      mooseError("Normalization postprocessor ", normpp, " requires execute_on = 'linear'");
```

Comment:
I am just curious:

We do not want to support regular Newton where "linear" might not make sense because we can not call residual evaluations at linear steps?

Reply:
Good point. In case of Newton, we will need to one residual evaluation to update this postprocessor.